### DISCUSSION TITLE
How to disable "ctrl+w ctrl+h", and other ...j, k, l mappings so I  could reliably navigate between groups with "ctrl+w ..." mappings?

### DISCUSSION THREAD

**Discussion by avimmer** [vscode-neovim/vscode-neovim#2202] [Q&A]
Hi, I'm having a hard time finding a keyboard binding which I believe was introduced by the extension. 
"ctrl+w h"
"ctrl+w j"
"ctrl+w k"
"ctrl+w l"
work as I expect by navigating to views. However, when I do that quick I end up pressing 
"ctrl+w ctrl+h"
"ctrl+w ctrl+j"
"ctrl+w ctrl+k"
"ctrl+w ctrl+l"
instead of navigating I receive something which resembles workbench.action.moveEditorToAboveGroup and so on. This behaviour is not like in neovim and is very irritating. 

Problem is, I cannot find any keyboard shortcut like "ctrl+w ctrl+h" and  workbench.action.moveEditorToAboveGroup aren't seem to be set. "ctrl+w ctrl+q" quits vscode instead of closing the view and I cannot find that in the list of keybindings either.
 Any idea where could I look those?

**Comment by jamestrew** [comment]
It looks like they are bound here https://github.com/vscode-neovim/vscode-neovim/blob/6a356a465770cba9d9b67deaceb38b6b19f3aee2/runtime/vscode/overrides/vscode-window-commands.vim#L91-L116

I was able to just override them in my init.lua
